tools/c7n_azure/tests/test_sqlserver.py

```diff
@@ -39,9 +39,9 @@ def test_find_by_name(self):
             'filters': [
                 {'type': 'value',
                  'key': 'name',
-                 'op': 'eq',
+                 'op': 'glob',
                  'value_type': 'normalize',
-                 'value': 'cctestsqlserver12262018'}],
+                 'value': 'cctestsqlserver*'}],
```

Comment:
are you changing this to globs because sqlserver names need to be unique?

Reply:
yes, similar to storage